test/factory.test.js

```diff
@@ -439,9 +400,9 @@ describe('Factory', function () {
 
         assume(factory.output).to.be.an('object');
         assume(factory.output['base.min.css'].content).to.be.instanceof(Buffer);
-        assume(factory.output['base.min.css'].content.toString()).to.include('span{margin:0;font-size:12px;color:#FFF}');
+        assume(factory.output['base.min.css'].content.toString()).to.include('span{margin:0;font-size:12px;color:#fff}');
```

Comment:
lol. Are lowercase `f`s smaller? 😂

Reply:
/shrug :) change brought in by clean-css@4, they might have reasons or just OCD